pkg/mount-manager/safe-mounter_windows.go

```diff
@@ -22,16 +22,18 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
-	diskv1alpha1 "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1alpha1"
-	diskclientv1alpha1 "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1alpha1"
+	diskv1beta1 "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1beta1"
+	diskclientv1beta1 "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1beta1"
 
 	fsv1alpha1 "github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1alpha1"
```

Comment:
These apis don't need to be updated to beta?

Reply:
so because filesystem and smb api groups do not have any changes since alpha, so we didn't move it to beta, considering they not as mature as others. We think the version of the software/code is independent of the version of the API groups.
 I discuss with @ddebroy and we will plan on moving them to beta too as post-beta action.

Reply:
From an API perspective, alpha has no guarantee of backwards compatibility or upgrade support. I would be wary of using an alpha api in production.

Reply:
right,  i agree. I think I should not say "they not as mature as others. " because there is no change. I support moving them to beta, the APIs are working fine with all the current drivers for windows. I will address it in csi-windows meetup.